src/test/bin/quic_gtest.cpp

```diff
@@ -985,9 +985,9 @@ int main(int argc, char** argv) {
     for (int i = 0; i < argc; ++i) {
         if (strcmp("--kernel", argv[i]) == 0) {
```

Comment:
Shouldn't you be checking for `--kernelPriv` here too?

Reply:
No need. `--kernel` strcmp will match `--kernelPriv`

Reply:
strcmp compares to the null terminator. They won't be equal.